src/components/molecules/Button.tsx

```diff
@@ -0,0 +1,72 @@
+import { css } from '@emotion/react'
+import React from 'react'
+
+import Icon from '../atoms/Icon'
+
+export type ButtonVariant = 'primary'
+
+/** BASIC BUTTON */
+export type ButtonProps = {
+  title?: string
+  variant?: ButtonVariant
+  size?: string
+  hasIcon?: boolean
```

Comment:
```suggestion
  icon?: ReactNode
```
처럼 설계하면 추후 다양한 아이콘들을 받을 수 있어서 좋을 것 같습니다!

```
<Button icon={<Icon name="arrow" />}>버튼이에용</Button>
```

Reply:
https://github.com/Lubycon/clelab-frontend/pull/7/commits/3cc6377a7c30fd1c0fb15304cdeeee5233402de8 고정이 되어 버렸네요